aksel.nav.no/website/components/website-modules/utils/amplitude.tsx

```diff
@@ -98,9 +108,44 @@ export const usePageView = (router: Router, pageProps: any) => {
       side: window.location.pathname,
       prosent: scrollD,
     });
-  }, []);
+
+    try {
+      if (!(isDevelopment || isTest || isPreview()) && hasMetrics) {
+        const { metrics } = pageProps.page;
+        const _id = pageProps?.page._id;
```

Comment:
Er ikke garantert at en side har et `page`-object med `_id`, eksempelvis temasider 

<img width="376" alt="Screenshot 2023-01-10 at 11 49 17" src="https://user-images.githubusercontent.com/26967723/211531923-d8e64fd2-9088-4027-a2ae-28af663588ad.png">

Men alle sider skal ha en `id`, så kan da bare bruke `pageProps.id` for å fikse det 🚀